.expeditor/verify.pipeline.yml

```diff
@@ -587,6 +587,34 @@ steps:
       - gopath-checkout#v1.0.1:
           import: github.com/chef/automate
 
+  - label: "compliance-service scanner with inspec master"
+    command:
+      - sudo sysctl -w vm.max_map_count=262144 # Needed for ElasticSearch 6
+      - export PATH=\$PATH:/usr/local/go/bin:\$HOME/.local/bin:\$HOME/bin
+      - sudo scripts/install_inspec_master.sh
+      - sudo scripts/install_golang.sh
+      - sudo scripts/install_grpcurl.sh
+      - cd components/compliance-service
+      - sudo -E  PATH=\$(hab pkg path core/ruby)/bin:\$PATH make test-integration-scanner
+      - cd ../.. && sudo git clean -fxd
+    artifact_paths:
+      - /tmp/secrets-service.log
+      - /tmp/event-service.log
+      - /tmp/compliance.log
+      - /tmp/nodemanager-service.log
+    env:
+      ES_VER: 6
+      LANG: en_US.UTF-8
+    timeout_in_minutes: 20
+    expeditor:
+      executor:
+        linux:
+          single-use: true
+          privileged: true
+    plugins:
+      - gopath-checkout#v1.0.1:
+          import: github.com/chef/automate
```

Comment:
This is a copy of the existing task for the scanner tests that calls a different script to install inspec.